### ISSUE TITLE
reckless unit tests failing in VLS project CI

### ISSUE THREAD

**Issue by ksedgwic** [ElementsProject/lightning#6198]
### Issue and Steps to Reproduce
The tests
- `tests/test_reckless.py::test_disable_enable`
- `tests/test_reckless.py::test_install`

Fail in VLS CI (basically CLN integration with modifications to use VLS remote signing daemon):
```
--------------------------- Captured stderr teardown ---------------------------
DEBUG:root:Test execution failed, leaving the test directory /tmp/ltests-usn1i3aa/test_disable_enable_1 intact.
_________________________________ test_install _________________________________
[gw2] linux -- Python 3.10.6 /usr/bin/python3
node_factory = <pyln.testing.utils.NodeFactory object at 0x7f0f1a7adc90>
    def test_install(node_factory):
        """test search, git clone, and installation to folder."""
        n = get_reckless_node(node_factory)
        r = reckless([f"--network={NETWORK}", "-v", "install", "testplugpass"], dir=n.lightning_dir)
        assert r.returncode == 0
        assert 'dependencies installed successfully' in r.stdout
        assert 'plugin installed:' in r.stdout
        assert 'testplugpass enabled' in r.stdout
>       check_stderr(r.stderr)
tests/test_reckless.py:156: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
stderr = "WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv\n"
    def check_stderr(stderr):
        def output_okay(out):
            for warning in ['[notice]', 'npm WARN', 'npm notice']:
                if out.startswith(warning):
                    return True
            return False
        for e in stderr.splitlines():
            if len(e) < 1:
                continue
            # Don't err on verbosity from pip, npm
>           assert output_okay(e)
E           assert False
E            +  where False = <function check_stderr.<locals>.output_okay at 0x7f0f1b363e20>("WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv")
tests/test_reckless.py:112: AssertionError
---------------------------- Captured stdout setup -----------------------------
```
Log of failing run in VLS CI:
https://gitlab.com/lightning-signer/vls-hsmd/-/jobs/4158759865

### `getinfo` output
```
./lightningd/lightningd --version
v23.05rc1-46-g2bdade5
```

**Comment by ksedgwic** [comment]
Might be that we need to alter/improve our CI run somehow?  Please advise ...

**Comment by ksedgwic** [comment]
Confirmed that this is fixed w/ 782c179

**Comment by vincenzopalazzo** [comment]
thanks @ksedgwic
